internal/cli/cmd/approve/approve.go

```diff
@@ -33,10 +33,10 @@ func NewCommand(cfg config.CLIConfig, opt *option.Option) *cobra.Command {
 		Short: "Manually approve freight for promotion to a stage",
 		Args:  option.NoArgs,
 		Example: `
-# Approve a freight in a project to the QA stage
+# Approve a piece of freight for the QA stage
 kargo approve --project=my-project --freight=abc1234 --stage=qa
 
-# Approve a freight in the default project to the QA stage
+# Approve a piece of freight for the QA stage in the default project
```

Comment:
In many cases, I am wondering about the necessity of the second example for the default project.

I have no strong opinion about it, but think it's something that is worth some consideration.

Any thoughts?

Reply:
Well, personally I only found out about this setting once I had seen it in one of the commands where it was documented. So I figured I would add it everywhere for now, until we have extensive documentation in other places.